lib/matplotlib/backends/backend_qt5.py

```diff
@@ -493,8 +497,7 @@ def notify_axes_change(fig):
 
     @QtCore.Slot()
     def _show_message(self, s):
-        # Fixes a PySide segfault.
```

Comment:
Looking at the history it seems that connecting the showMessage slot directly resulted in a segfault in PySide. The `_show_message` wrapper appears to be a workaround.